src/main/java/wolox/reactortraining/models/Topic.java

```diff
@@ -0,0 +1,34 @@
+package wolox.reactortraining.models;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+import org.springframework.data.mongodb.core.mapping.Document;
+import wolox.reactortraining.dtos.TopicDto;
+
+@Document
+@Getter
+@NoArgsConstructor
+@Setter
+@ToString
+public class Topic {
+
+    @Id
+    private String id;
+
+    private String description;
+
+    @DBRef
+    private List<User> users;
+
+    public static Topic fromDto(TopicDto dto) {
```

Comment:
remove this static method, this conversion from dto to entity make it in the constructor